scripts/gen-i18n.ts

```diff
@@ -142,36 +149,35 @@ function getTranslationsJs(file) {
             tokens: true,
             plugins,
         });
-        traverse.default(babelParsed, {
+        traverse(babelParsed, {
             enter: (p) => {
-                const node = p.node;
-                if (p.isCallExpression() && node.callee && TRANSLATIONS_FUNCS.includes(node.callee.name)) {
-                    const tKey = getTKey(node.arguments[0]);
+                if (isCallExpression(p.node) && isIdentifier(p.node.callee) && TRANSLATIONS_FUNCS.includes(p.node.callee.name)) {
+                    const tKey = getTKey(p.node.arguments[0]);
 
                     // This happens whenever we call _t with non-literals (ie. whenever we've
                     // had to use a _td to compensate) so is expected.
                     if (tKey === null) return;
 
                     // check the format string against the args
                     // We only check _t: _td has no args
-                    if (node.callee.name === '_t') {
+                    if (isIdentifier(p.node.callee) && p.node.callee.name === '_t') {
                         try {
                             const placeholders = getFormatStrings(tKey);
-                            for (const placeholder of placeholders) {
-                                if (node.arguments.length < 2) {
+                            for (const placeholder of Array.from(placeholders)) {
```

Comment:
I don't think the `Array.from` should be necessary, a for can take any iterable

Reply:
Just my IDE is unhappy, tsc seems happy